### PR TITLE
Add minio-operator package

### PR DESCRIPTION
**Pull request by jamonation**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)
- [ ] If non-streamed package names no longer built, open PR to withdraw them (see [WITHDRAWING PACKAGES](https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md))

#### For package updates (renames) in the base images

When updating packages part of base images (i.e. cgr.dev/chainguard/wolfi-base or ghcr.io/wolfi-dev/sdk)
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk images successfully build
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk contain no obsolete (no longer built) packages
- [ ] Upon launch, does `apk upgrade --latest` successfully upgrades packages or performs no actions

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: minio-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      mkdir ${{targets.destdir}}/licenses
+      cp CREDITS LICENSE ${{targets.destdir}}/licenses/
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: compatibility symlinks package for minio-operator Dockerfile
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/
+          ln -sf /usr/bin/minio-operator ${{targets.contextdir}}/minio-operator
```
Comment (jamonation) [inline minio-operator.yaml:48]: Could this be part of the main package to simplify things a bit?

Comment (developer-guy) [reply minio-operator.yaml:48]: we have been using compat packages for these symlinks as you know, its ok for me but I'm not sure we should break the standard here?

Comment (jamonation) [reply minio-operator.yaml:48]: all good, works for me!

**Review by developer-guy — APPROVED** [review]

**Review by mamccorm — DISMISSED** [review]
This one looks to be version streamed:
 - https://github.com/minio/operator/tags

We'll need to change package naming approach etc here

**Review by EyeCantCU — DISMISSED** [review]
The controller will not work as intended unless we generate the code used by it: https://github.com/minio/operator/blob/master/k8s/update-codegen.sh

**Review by EyeCantCU — COMMENTED** [review]
Actually, it looks like they already bundle it in the source

**Comment by EyeCantCU** [comment]
@mamccorm that last release would suggest they support streams, but if you look at past releases there isn't a case I can find where they've released an update for a prior version before. They also do not have any documentation on the versions of the operator they actively update. Everything I've seen would suggest they're only supporting the latest release

**Comment by developer-guy** [comment]
so are we going to merge this guys?

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**